app/app/vendor/routes/index.js

```diff
@@ -0,0 +1,12 @@
+exports.home = {
+  templateUrl: '/templates/layout/_body.html'
+};
+
+exports.permission_denied = {
```

Comment:
Identifier 'permission_denied' is not in camel case.<br>'exports' is not defined.